site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -53,14 +56,16 @@ include::docs/src/json/identity-providers/links/create-link-request.json[]
 === Response
 
 :never_search_error:
+:webhook_event:
 include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:webhook_event!:
 :never_search_error!:
 
 include::docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc[]
 
 == Complete a pending Link
 
-This API is used complete a pending Link. If an identity provider is configured with a linking strategy of `Create a pending link`, a pending link Id will be returned from the Login API that can be used here.
+This API is used complete a pending link. If an identity provider is configured with a linking strategy of `Create a pending link`, a `pendingLinkId` will be returned from the link:/docs/v1/tech/apis/identity-providers/#complete-the-login[Login API].  This value can be used in the below request.
```

Comment:
Ah, I see why you added the stub.  Not sure the stub makes sense - as it is a duplicate, and not a complete API doc.  Maybe this just says it is returned by the Identity Provider Login API (see the Complete Login) for the respective IdP - or something like that. 


Suggestion:

>  This value can be used in the request below.